lib/Chunk.js

```diff
@@ -126,7 +127,7 @@ class Chunk {
 
 		/** @private */
 		this._groups = new SortableSet(undefined, sortChunkGroupById);
-		/** @private @type {SortableSet<ChunkGroup>} */
+		/** @private @type {SortableSet<ChunkGroup | Entrypoint>} */
```

Comment:
Entrypoint extends ChunkGroup. This should not be needed.

Reply:
Actually we can drop the `Entrypoint`.